addons/payment_transfer/views/payment_views.xml

```diff
@@ -11,7 +11,7 @@
             </field>
             <xpath expr="//group[@name='payment_followup']" position="attributes">
                 <attribute name="attrs">
-                    {'invisible': [('provider', '=', 'transfer')]}
+                    {'invisible': [('provider', 'not in', ['transfer', 'test'])]}
```

Comment:
Not sure about this, the user would not be able to change the journal on any payment acquirer except for transfer and test one